.github/workflows/build.yml

```diff
@@ -0,0 +1,13 @@
+name: Build container
```

Comment:
We do not need this GitHub Actions pipeline. 

Sorry that I didn't explain clearly on this. I have a dedicated GitHub repo [legendu-net/docker_image_builder](https://github.com/legendu-net/docker_image_builder) which handles building of all my Docker images. It has several advantages:
1. Auto parse the graph dependency and build Docker images in the right order. All image/tag combinations are built exactly once
2. Better build caching and lower pipeline latency since all images are built on the same VM. It usually takes less than 2 hours to build all my images for all tags (branches). This is much faster than the free service provided by DockerHub which can take days and even up to a week. 
3. Robust build pipeline. If an intermediate image `dclong/some_image:some_tag` fails to build, the whole pipeline will continue build using the last version of `dclong/some_image:some_tag`. Of course, images failed to build will be reported after the pipeline finishes. 

I've created the following PR to build Docker images based on the `alpine` branch.  
https://github.com/legendu-net/docker_image_builder/pull/16